dluhc-component-library/src/components/maps/DrawingLayer.tsx

```diff
@@ -1,9 +1,12 @@
-import Polygon from "ol/geom/Polygon";
+import { Feature } from "ol";
+import { Polygon } from "ol/geom";
 import { Draw } from "ol/interaction";
+import { DrawEvent } from "ol/interaction/Draw";
 import VectorLayer from "ol/layer/Vector";
-import VectorSource, { VectorSourceEvent } from "ol/source/Vector";
-import { useEffect, useRef } from "react";
+import VectorSource from "ol/source/Vector";
+import { useEffect, useRef } from "preact/compat";
 import { useMap } from "src/contexts/mapContext";
+import { Polygon as Boundary } from "./types";
```

Comment:
I dont think we should be renaming types like this, i think it could get very confusing when you look at what props you should be passing.

We should stick to the word Polygon or Boundary not use them interchangeably.

Reply:
Yeah, I though Polygon was the most generic name, but then it clashed with the Open Layers `Polygon` class. I've renamed it to `Boundary` since that's the only use for it as a type at the moment.